participant/admin.py

```diff
@@ -2,4 +2,8 @@
 from .models import Participant
 
 
-admin.site.register(Participant)
+class ParticipantList(admin.ModelAdmin):
+    list_filter = ("exercise",)
+
+
+admin.site.register(Participant, ParticipantList)
```

Comment:
Can we use the decorator instead to register the model? Its nicer and more consistent with all our admin model registration behaviour.